docs/extensions/jvm.md

```diff
@@ -101,3 +101,31 @@ export interface JvmRunEnvironmentResult{
     environmentVariables: Map<String, String>;
 }
 ```
+
+### JVM Dependency Module
```

Comment:
I think this extension should be called `Maven Dependency Module` because:
- there are other kinds of dependencies with slightly different concepts (Ivy contains `extensions` but Maven does not)
- Maven Dependency Modules can also be used for Scala.js

Reply:
I expected that there will be no difference between Maven/Ivy dependencies.
It's more like `ModuleID` + resolution data in sbt terms.

Reply:
sbt `ModuleId` looks simple but it is more complex than Maven or Ivy modules: https://github.com/sbt/librarymanagement/blob/23b7d923af2e93d5eebd67c93352758160462388/core/src/main/contraband/librarymanagement.json#L335-L353

Reply:
I want to call it `MavenModuleId` because it is the simplest one that is used in the JVM ecosystem. It does not imply that all dependencies are coming from Maven repositories.
Then if we want to add support for `Scala.js` we can reuse it.We can also add a `NpmModuleId` for native js libraries.

Reply:
> It does not imply that all dependencies are coming from Maven repositories.

Oh ok, I got what you mean. Will rename it!

Reply:
Done